docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -81,6 +81,39 @@ The path to the Teku data directory. The default directory is OS dependent:
 
 The default Docker image location is `/root/.local/share/teku`.
 
+### data-storage-archive-frequency
+
+```bash tab="Syntax"
+--data-storage-archive-frequency=<NUMBER>
+```
+
+```bash tab="Command Line"
+--data-storage-archive-frequency=1028
+```
+
+```bash tab="Environment Variable"
+TEKU_DATA_STORAGE_ARCHIVE_FREQUENCY=1028
+```
+
+```bash tab="Configuration File"
+data-storage-archive-frequency: 1028
+```
+
+Set the frequency (in slots) at which to store finalized states to disk. Defaults to 2048.
+
+This option is ignored if [`--data-storage-mode`](#data-storage-mode) is set to `prune`.
+
+!!! note
+    Lower archive frequencies (saved finalized states with a greater difference between slot counts)
```

Comment:
I think it might be clearer to refer to the size of the number.  Maybe: 

Specifying a larger number of slots as the archive frequency has a potentially higher overhead for retrieving finalized states since more states may need to be regenerated to get to the requested state. Specifying a lower number of slots as the frequency increases the disk space usage.